emcc.py

```diff
@@ -2139,6 +2139,12 @@ def dedup_list(lst):
   # specified by the user
   shared.Settings.EXPORTED_FUNCTIONS = dedup_list(shared.Settings.EXPORTED_FUNCTIONS)
 
+  if options.oformat == OFormat.WASM:
+    # If we are outputing directly to wasm we won't be running the JS compiler post
```

Comment:
Is this true? Maybe I'm confused but I think we still run `post_link` and get up to `emscripten.run`. It's only after that that we reach
```python
if options.oformat == OFormat.WASM:
    return
```

Reply:
Ah.. but emscripten.run has an early return in it... I can implement this instead by moving code around there maybe.  Let me try that less invasive approach.